src/Compilers/VisualBasic/Test/Emit/Emit/CompilationEmitTests.vb

```diff
@@ -2923,5 +2923,89 @@ End Class
             End Using
         End Sub
 
+        Private Class FailingStream
```

Comment:
`FailingStream` and `BrokenStream` seem pretty similar -- what's the reasoning about having two different streams?